pkg/restic/restic.go

```diff
@@ -306,6 +306,7 @@ func (r *Repo) ListDirectory(ctx context.Context, snapshot string, path string,
 		// an empty path can trigger very expensive operations (e.g. iterates all files in the snapshot)
 		return nil, nil, errors.New("path must not be empty")
 	}
+	path = toPathFilter(path)
```

Comment:
Hey, can you elaborate a bit on what motivated this change? Is this for test coverage or is there also an issue affecting browsing snapshots on windows?

Reply:
Thanks for checking that on that! I was skeptical this was a real bug, since I hadn't seen errors browsing on Windows. Will update in a bit.